File-Formats/PDF/Convert-HTML-To-PDF/net-core.md

```diff
@@ -11,63 +11,63 @@ keywords: Assemblies
 
 The Syncfusion HTML to PDF converter is a .NET library used to convert HTML or web pages to PDF. Using this library you can convert HTML to PDF in ASP.NET Core application.  
 
-## Steps to convert HTML to PDF in ASP.NET Core application:
+## Steps to convert HTML to PDF in ASP.NET Core application
 
-Create a new C# ASP.NET Core Web Application project.
+1. Create a new C# ASP.NET Core Web Application project.
 ![convert_HtmltoPdf_ASPNET_CORE1](htmlconversion_images/aspnetcore1.png)
 
-Install Syncfusion.HtmlToPdfConverter.Net.Windows NuGet package as reference to your .NET Standard applications from NuGet.org.
+2. Install [Syncfusion.HtmlToPdfConverter.Net.Windows](https://www.nuget.org/packages/Syncfusion.HtmlToPdfConverter.Net.Windows) NuGet package as reference to your .NET Standard applications from [NuGet.org](https://www.nuget.org/).
 ![convert_HtmltoPdf_ASPNET_CORE2](htmlconversion_images/aspnetcore2.png)
 
-A default controller with name HomeController.cs gets added on creation of ASP.NET Core MVC project. Include the following namespaces in that HomeController.cs file.
+3. A default controller with name HomeController.cs gets added on creation of ASP.NET Core MVC project. Include the following namespaces in that HomeController.cs file.
 
-{% highlight c# tabtitle="C#" %}
+   {% highlight c# tabtitle="C#" %}
 
-using Syncfusion.Pdf;
-using Syncfusion.HtmlConverter;
+   using Syncfusion.Pdf;
+   using Syncfusion.HtmlConverter;
+   using System.IO;
 
-{% endhighlight %}
+   {% endhighlight %}
 
-Add an Export To PDF button in index.cshtml as shown below.
+4. Add an Export To PDF button in index.cshtml as shown below.
```

Comment:
Add a new button in index.cshtml as shown below.

Reply:
content modified